static-analysis/autodispose-lint/src/main/kotlin/com/uber/autodispose/lint/AutoDisposeDetector.kt

```diff
@@ -107,8 +108,8 @@ class AutoDisposeDetector: Detector(), SourceCodeScanner {
   private var lenient: Boolean = false
 
   override fun beforeCheckRootProject(context: Context) {
-    // Add the default Android scopes.
-    val scopes = HashSet(DEFAULT_SCOPES)
+    var overrideScopes = false
+    val scopes = HashSet<String>()
```

Comment:
nit: can we make this a more idiomatic `mutableSetOf()` here while we're at it?

Reply:
d476f27e99b04f6d1fe8b4c782914292811689a6